In [9]:
import pandas as pd

food_insec = pd.read_csv('MMG2025_2019-2023_Data_To_Share_County_CSV.csv', encoding='latin-1')

print(food_insec)

        FIPS State            County, State  Year  \
0       1001    AL  Autauga County, Alabama  2019   
1       1001    AL  Autauga County, Alabama  2020   
2       1001    AL  Autauga County, Alabama  2021   
3       1001    AL  Autauga County, Alabama  2022   
4       1001    AL  Autauga County, Alabama  2023   
...      ...   ...                      ...   ...   
15711  56045    WY   Weston County, Wyoming  2019   
15712  56045    WY   Weston County, Wyoming  2020   
15713  56045    WY   Weston County, Wyoming  2021   
15714  56045    WY   Weston County, Wyoming  2022   
15715  56045    WY   Weston County, Wyoming  2023   

      Overall Food Insecurity Rate # of Food Insecure Persons Overall  \
0                            15.7%                              8,670   
1                            14.5%                              8,070   
2                            13.3%                              7,770   
3                            15.1%                              8,860  

In [10]:
food_insec.head()

,FIPS,State,"County, State",Year,Overall Food Insecurity Rate,# of Food Insecure Persons Overall,Food Insecurity Rate among Black Persons (all ethnicities),Food Insecurity Rate among Hispanic Persons (any race),"Food Insecurity Rate among White, non-Hispanic Persons",SNAP Threshold,...,% food insecure children in HH w/ HH incomes below 185 FPL,% food insecure children in HH w/ HH incomes above 185 FPL,Cost Per Meal,Weighted weekly $ needed by FI,Weighted Annual Food Budget Shortfall,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023),Census Region,Census Division,FNS Region
0,1001,AL,"Autauga County, Alabama",2019,15.7%,"8,670",26%,NaN,12%,130%,...,69%,31%,$3.00,$16.88,"$4,439,000",2.0,NaN,South,East South Central,Southeast
1,1001,AL,"Autauga County, Alabama",2020,14.5%,"8,070",25%,NaN,9%,130%,...,71%,29%,$3.22,$17.09,"$4,184,000",2.0,NaN,South,East South Central,Southeast
2,1001,AL,"Autauga County, Alabama",2021,13.3%,"7,770",23%,NaN,9%,130%,...,68%,32%,$3.60,$20.99,"$4,947,000",2.0,NaN,South,East South Central,Southeast
3,1001,AL,"Autauga County, Alabama",2022,15.1%,"8,860",23%,NaN,11%,130%,...,68%,32%,$4.01,$24.86,"$6,680,000",2.0,2.0,South,East South Central,Southeast
4,1001,AL,"Autauga County, Alabama",2023,15.1%,"8,970",24%,NaN,12%,130%,...,66%,34%,$3.64,$22.72,"$6,183,000",2.0,2.0,South,East South Central,Southeast


In [11]:
food_insec.info()


<class 'pandas.DataFrame'>
RangeIndex: 15716 entries, 0 to 15715
Data columns (total 24 columns):
 #   Column                                                      Non-Null Count  Dtype  
---  ------                                                      --------------  -----  
 0   FIPS                                                        15716 non-null  int64  
 1   State                                                       15716 non-null  str    
 2   County, State                                               15716 non-null  str    
 3   Year                                                        15716 non-null  int64  
 4   Overall Food Insecurity Rate                                15715 non-null  str    
 5   # of Food Insecure Persons Overall                          15715 non-null  str    
 6   Food Insecurity Rate among Black Persons (all ethnicities)  7432 non-null   str    
 7   Food Insecurity Rate among Hispanic Persons (any race)      8915 non-null   str    
 8   Food In

In [12]:
food_insec.shape

(15716, 24)

In [18]:
food_insec['County, State'].value_counts()

County, State
Autauga County, Alabama                                   5
Baldwin County, Alabama                                   5
Barbour County, Alabama                                   5
Bibb County, Alabama                                      5
Blount County, Alabama                                    5
                                                         ..
South Central Connecticut Planning Region, Connecticut    2
Southeastern Connecticut Planning Region, Connecticut     2
Western Connecticut Planning Region, Connecticut          2
Doña Ana County, New Mexico                               2
Valdez-Cordova Census Area, Alaska                        1
Name: count, Length: 3154, dtype: int64

In [19]:
food_insec['State'].value_counts()

State
TX    1270
GA     795
VA     665
KY     600
MO     575
KS     525
IL     510
NC     500
IA     495
TN     475
NE     465
IN     460
OH     440
MN     435
MI     415
MS     410
OK     385
AR     375
WI     360
AL     335
FL     335
PA     335
SD     330
CO     320
LA     320
NY     310
CA     290
MT     280
WV     275
ND     265
SC     230
ID     220
WA     195
OR     180
NM     165
AK     149
UT     145
MD     120
WY     115
NJ     105
NV      85
ME      80
AZ      75
MA      70
VT      70
NH      50
CT      42
HI      25
RI      25
DE      15
DC       5
Name: count, dtype: int64

In [14]:
food_insec.describe()

,FIPS,Year,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023)
count,15716.000000,15716.000000,15694.000000,6288.000000
mean,30373.833927,2021.000318,5.011979,5.249046
std,15166.194087,1.414236,2.707610,2.929423
min,1001.000000,2019.000000,1.000000,1.000000
25%,18175.000000,2020.000000,2.000000,2.000000
50%,29175.000000,2021.000000,6.000000,6.000000
75%,45081.000000,2022.000000,7.000000,8.000000
max,56045.000000,2023.000000,9.000000,9.000000


In [32]:
food_insec.loc[4971:5570]

,FIPS,State,"County, State",Year,Overall Food Insecurity Rate,# of Food Insecure Persons Overall,Food Insecurity Rate among Black Persons (all ethnicities),Food Insecurity Rate among Hispanic Persons (any race),"Food Insecurity Rate among White, non-Hispanic Persons",SNAP Threshold,...,% food insecure children in HH w/ HH incomes below 185 FPL,% food insecure children in HH w/ HH incomes above 185 FPL,Cost Per Meal,Weighted weekly $ needed by FI,Weighted Annual Food Budget Shortfall,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023),Census Region,Census Division,FNS Region
4971,21001,KY,"Adair County, Kentucky",2019,16.7%,"3,210",20%,NaN,16%,200%,...,78%,22%,$2.78,$15.69,"$1,527,000",7.0,NaN,South,East South Central,Southeast
4972,21001,KY,"Adair County, Kentucky",2020,15.2%,"2,950",17%,NaN,14%,200%,...,79%,21%,$2.79,$14.83,"$1,327,000",7.0,NaN,South,East South Central,Southeast
4973,21001,KY,"Adair County, Kentucky",2021,13.7%,"2,590",NaN,NaN,14%,200%,...,82%,19%,$3.04,$17.69,"$1,390,000",7.0,NaN,South,East South Central,Southeast
4974,21001,KY,"Adair County, Kentucky",2022,18.0%,"3,400",NaN,NaN,17%,200%,...,82%,18%,$3.40,$21.07,"$2,173,000",7.0,7.0,South,East South Central,Southeast
4975,21001,KY,"Adair County, Kentucky",2023,18.5%,"3,530",NaN,NaN,18%,200%,...,89%,11%,$3.16,$19.72,"$2,111,000",7.0,7.0,South,East South Central,Southeast
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
5566,21239,KY,"Woodford County, Kentucky",2019,11.2%,"2,960",16%,NaN,12%,200%,...,65%,35%,$3.29,$18.53,"$1,664,000",2.0,NaN,South,East South Central,Southeast
5567,21239,KY,"Woodford County, Kentucky",2020,9.9%,"2,620",18%,NaN,10%,200%,...,74%,26%,$3.32,$17.62,"$1,400,000",2.0,NaN,South,East South Central,Southeast
5568,21239,KY,"Woodford County, Kentucky",2021,8.0%,"2,150",12%,NaN,9%,200%,...,61%,39%,$3.58,$20.83,"$1,358,000",2.0,NaN,South,East South Central,Southeast
5569,21239,KY,"Woodford County, Kentucky",2022,11.7%,"3,150",12%,NaN,11%,200%,...,57%,43%,$4.03,$24.95,"$2,384,000",2.0,2.0,South,East South Central,Southeast


In [26]:
food_insec.loc[1124:1128]

,FIPS,State,"County, State",Year,Overall Food Insecurity Rate,# of Food Insecure Persons Overall,Food Insecurity Rate among Black Persons (all ethnicities),Food Insecurity Rate among Hispanic Persons (any race),"Food Insecurity Rate among White, non-Hispanic Persons",SNAP Threshold,...,% food insecure children in HH w/ HH incomes below 185 FPL,% food insecure children in HH w/ HH incomes above 185 FPL,Cost Per Meal,Weighted weekly $ needed by FI,Weighted Annual Food Budget Shortfall,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023),Census Region,Census Division,FNS Region
1124,6077,CA,"San Joaquin County, California",2019,11.5%,"85,740",20%,16%,9%,200%,...,71%,29%,$3.20,$18.03,"$46,879,000",2.0,NaN,West,Pacific,Western
1125,6077,CA,"San Joaquin County, California",2020,12.1%,"90,820",20%,16%,7%,200%,...,71%,29%,$3.53,$18.72,"$51,570,000",2.0,NaN,West,Pacific,Western
1126,6077,CA,"San Joaquin County, California",2021,10.8%,"83,690",18%,13%,7%,200%,...,68%,32%,$3.92,$22.80,"$57,892,000",2.0,NaN,West,Pacific,Western
1127,6077,CA,"San Joaquin County, California",2022,12.7%,"98,860",23%,19%,9%,200%,...,65%,35%,$4.28,$26.53,"$79,562,000",2.0,2.0,West,Pacific,Western
1128,6077,CA,"San Joaquin County, California",2023,13.6%,"107,140",24%,21%,10%,200%,...,67%,33%,$3.77,$23.54,"$76,505,000",2.0,2.0,West,Pacific,Western


In [29]:
food_insec.loc[15231:15235]

,FIPS,State,"County, State",Year,Overall Food Insecurity Rate,# of Food Insecure Persons Overall,Food Insecurity Rate among Black Persons (all ethnicities),Food Insecurity Rate among Hispanic Persons (any race),"Food Insecurity Rate among White, non-Hispanic Persons",SNAP Threshold,...,% food insecure children in HH w/ HH incomes below 185 FPL,% food insecure children in HH w/ HH incomes above 185 FPL,Cost Per Meal,Weighted weekly $ needed by FI,Weighted Annual Food Budget Shortfall,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023),Census Region,Census Division,FNS Region
15231,54107,WV,"Wood County, West Virginia",2019,13.4%,"11,420",28%,NaN,13%,200%,...,75%,25%,$2.78,$15.68,"$5,430,000",3.0,NaN,South,South Atlantic,Mid-Atlantic
15232,54107,WV,"Wood County, West Virginia",2020,12.8%,"10,830",29%,NaN,11%,200%,...,78%,22%,$2.85,$15.11,"$4,965,000",3.0,NaN,South,South Atlantic,Mid-Atlantic
15233,54107,WV,"Wood County, West Virginia",2021,10.8%,"9,140",28%,NaN,11%,200%,...,77%,24%,$3.34,$19.43,"$5,386,000",3.0,NaN,South,South Atlantic,Mid-Atlantic
15234,54107,WV,"Wood County, West Virginia",2022,13.6%,"11,480",NaN,NaN,14%,200%,...,74%,26%,$3.68,$22.79,"$7,934,000",3.0,3.0,South,South Atlantic,Mid-Atlantic
15235,54107,WV,"Wood County, West Virginia",2023,14.7%,"12,340",NaN,NaN,15%,200%,...,75%,25%,$3.31,$20.65,"$7,731,000",3.0,3.0,South,South Atlantic,Mid-Atlantic


In [30]:
food_insec.loc[14971:14975]

,FIPS,State,"County, State",Year,Overall Food Insecurity Rate,# of Food Insecure Persons Overall,Food Insecurity Rate among Black Persons (all ethnicities),Food Insecurity Rate among Hispanic Persons (any race),"Food Insecurity Rate among White, non-Hispanic Persons",SNAP Threshold,...,% food insecure children in HH w/ HH incomes below 185 FPL,% food insecure children in HH w/ HH incomes above 185 FPL,Cost Per Meal,Weighted weekly $ needed by FI,Weighted Annual Food Budget Shortfall,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023),Census Region,Census Division,FNS Region
14971,54003,WV,"Berkeley County, West Virginia",2019,9.2%,"10,580",18%,NaN,11%,200%,...,61%,39%,$2.96,$16.66,"$5,346,000",2.0,NaN,South,South Atlantic,Mid-Atlantic
14972,54003,WV,"Berkeley County, West Virginia",2020,8.3%,"9,730",23%,NaN,8%,200%,...,72%,28%,$3.19,$16.95,"$5,003,000",2.0,NaN,South,South Atlantic,Mid-Atlantic
14973,54003,WV,"Berkeley County, West Virginia",2021,7.2%,"8,710",20%,NaN,9%,200%,...,71%,29%,$3.56,$20.73,"$5,478,000",2.0,NaN,South,South Atlantic,Mid-Atlantic
14974,54003,WV,"Berkeley County, West Virginia",2022,10.2%,"12,620",23%,NaN,11%,200%,...,67%,33%,$3.91,$24.23,"$9,277,000",2.0,2.0,South,South Atlantic,Mid-Atlantic
14975,54003,WV,"Berkeley County, West Virginia",2023,11.7%,"14,810",25%,NaN,12%,200%,...,67%,33%,$3.52,$21.98,"$9,873,000",2.0,2.0,South,South Atlantic,Mid-Atlantic


In [31]:
food_insec.loc[14901:14905]

,FIPS,State,"County, State",Year,Overall Food Insecurity Rate,# of Food Insecure Persons Overall,Food Insecurity Rate among Black Persons (all ethnicities),Food Insecurity Rate among Hispanic Persons (any race),"Food Insecurity Rate among White, non-Hispanic Persons",SNAP Threshold,...,% food insecure children in HH w/ HH incomes below 185 FPL,% food insecure children in HH w/ HH incomes above 185 FPL,Cost Per Meal,Weighted weekly $ needed by FI,Weighted Annual Food Budget Shortfall,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023),Census Region,Census Division,FNS Region
14901,53053,WA,"Pierce County, Washington",2019,10.9%,"95,250",21%,18%,10%,200%,...,56%,44%,$3.40,$19.15,"$55,333,000",1.0,NaN,West,Pacific,Western
14902,53053,WA,"Pierce County, Washington",2020,10.6%,"94,860",20%,19%,7%,200%,...,55%,45%,$3.69,$19.60,"$56,395,000",1.0,NaN,West,Pacific,Western
14903,53053,WA,"Pierce County, Washington",2021,8.4%,"76,060",20%,16%,7%,200%,...,57%,43%,$4.11,$23.92,"$55,185,000",1.0,NaN,West,Pacific,Western
14904,53053,WA,"Pierce County, Washington",2022,11.2%,"102,530",24%,22%,9%,200%,...,51%,49%,$4.54,$28.14,"$87,502,000",1.0,1.0,West,Pacific,Western
14905,53053,WA,"Pierce County, Washington",2023,12.7%,"117,450",26%,23%,10%,200%,...,49%,51%,$3.99,$24.96,"$88,919,000",1.0,1.0,West,Pacific,Western


In [ ]:
food_insec.loc[964:968]

,FIPS,State,"County, State",Year,Overall Food Insecurity Rate,# of Food Insecure Persons Overall,Food Insecurity Rate among Black Persons (all ethnicities),Food Insecurity Rate among Hispanic Persons (any race),"Food Insecurity Rate among White, non-Hispanic Persons",SNAP Threshold,...,% food insecure children in HH w/ HH incomes below 185 FPL,% food insecure children in HH w/ HH incomes above 185 FPL,Cost Per Meal,Weighted weekly $ needed by FI,Weighted Annual Food Budget Shortfall,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023),Census Region,Census Division,FNS Region
964,6013,CA,"Contra Costa County, California",2019,7.6%,"87,180",15%,13%,5%,200%,...,56%,44%,$3.77,$21.27,"$56,245,000",1.0,NaN,West,Pacific,Western
965,6013,CA,"Contra Costa County, California",2020,8.5%,"97,160",15%,14%,3%,200%,...,56%,44%,$4.12,$21.88,"$64,482,000",1.0,NaN,West,Pacific,Western
966,6013,CA,"Contra Costa County, California",2021,7.5%,"87,070",14%,11%,4%,200%,...,60%,40%,$4.45,$25.92,"$68,469,000",1.0,NaN,West,Pacific,Western
967,6013,CA,"Contra Costa County, California",2022,9.2%,"107,300",19%,16%,6%,200%,...,53%,47%,$4.84,$29.97,"$97,544,000",1.0,1.0,West,Pacific,Western
968,6013,CA,"Contra Costa County, California",2023,10.3%,"119,690",21%,17%,7%,200%,...,53%,47%,$4.31,$26.95,"$97,836,000",1.0,1.0,West,Pacific,Western


In [15]:
food_insec.columns.tolist()

['FIPS',
 'State',
 'County, State',
 'Year',
 'Overall Food Insecurity Rate',
 '# of Food Insecure Persons Overall',
 'Food Insecurity Rate among Black Persons (all ethnicities)',
 'Food Insecurity Rate among Hispanic Persons (any race)',
 'Food Insecurity Rate among White, non-Hispanic Persons',
 'SNAP Threshold',
 '% FI ? SNAP Threshold',
 '% FI > SNAP Threshold',
 'Child Food Insecurity Rate',
 '# of Food Insecure Children',
 '% food insecure children in HH w/ HH incomes below 185 FPL',
 '% food insecure children in HH w/ HH incomes above 185 FPL',
 'Cost Per Meal',
 'Weighted weekly $ needed by FI',
 'Weighted Annual Food Budget Shortfall',
 'Rural-Urban Continuum Code (2013)',
 'Rural-Urban Continuum Code (2023)',
 'Census Region',
 'Census Division',
 'FNS Region']

In [35]:
food_insec.iloc[0:3, 0:2]

,FIPS,State
0,1001,AL
1,1001,AL
2,1001,AL


In [43]:
food_insec.sort_values(by=['Overall Food Insecurity Rate'], ascending=[True])

,FIPS,State,"County, State",Year,Overall Food Insecurity Rate,# of Food Insecure Persons Overall,Food Insecurity Rate among Black Persons (all ethnicities),Food Insecurity Rate among Hispanic Persons (any race),"Food Insecurity Rate among White, non-Hispanic Persons",SNAP Threshold,...,% food insecure children in HH w/ HH incomes below 185 FPL,% food insecure children in HH w/ HH incomes above 185 FPL,Cost Per Meal,Weighted weekly $ needed by FI,Weighted Annual Food Budget Shortfall,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023),Census Region,Census Division,FNS Region
2363,13171,GA,"Lamar County, Georgia",2021,10.0%,"1,850",16%,NaN,9%,130%,...,100%,0%,$3.11,$18.10,"$1,016,000",1.0,NaN,South,South Atlantic,Southeast
4409,19183,IA,"Washington County, Iowa",2022,10.0%,"2,260",NaN,16%,9%,160%,...,70%,30%,$3.82,$23.66,"$1,622,000",3.0,3.0,Midwest,West North Central,Midwest
10039,38033,ND,"Golden Valley County, North Dakota",2022,10.0%,170,NaN,NaN,8%,200%,...,72%,28%,$4.13,$25.60,"$132,000",9.0,9.0,Midwest,West North Central,Mountain Plains
8688,31165,NE,"Sioux County, Nebraska",2021,10.0%,110,NaN,NaN,9%,165%,...,100%,0%,$3.43,$19.99,"$67,000",9.0,NaN,Midwest,West North Central,Mountain Plains
1354,8051,CO,"Gunnison County, Colorado",2019,10.0%,"1,680",NaN,17%,10%,200%,...,96%,5%,$3.59,$20.21,"$1,030,000",7.0,NaN,West,Mountain,Mountain Plains
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
3350,17147,IL,"Piatt County, Illinois",2023,9.9%,"1,650",NaN,NaN,8%,165%,...,56%,44%,$3.63,$22.71,"$1,137,000",3.0,3.0,Midwest,East North Central,Midwest
14139,51013,VA,"Arlington County, Virginia",2022,9.9%,"23,390",24%,17%,6%,200%,...,91%,9%,$5.52,$34.23,"$24,287,000",1.0,1.0,South,South Atlantic,Mid-Atlantic
1269,8017,CO,"Cheyenne County, Colorado",2019,9.9%,200,NaN,NaN,10%,200%,...,86%,14%,$3.01,$16.96,"$103,000",9.0,NaN,West,Mountain,Mountain Plains
8188,30077,MT,"Powell County, Montana",2021,9.9%,690,NaN,NaN,9%,200%,...,71%,30%,$3.70,$21.52,"$451,000",6.0,NaN,West,Mountain,Mountain Plains


['State',
 'County, State',
 'Year',
 'Overall Food Insecurity Rate',
 '# of Food Insecure Persons Overall',
 'Food Insecurity Rate among Black Persons (all ethnicities)',
 'Food Insecurity Rate among Hispanic Persons (any race)',
 'Food Insecurity Rate among White, non-Hispanic Persons',
 'SNAP Threshold',
 '% FI ? SNAP Threshold',
 '% FI > SNAP Threshold',
 'Child Food Insecurity Rate',
 '# of Food Insecure Children',
 '% food insecure children in HH w/ HH incomes below 185 FPL',
 '% food insecure children in HH w/ HH incomes above 185 FPL',
 'Cost Per Meal',
 'Weighted weekly $ needed by FI',
 'Weighted Annual Food Budget Shortfall',
 'Rural-Urban Continuum Code (2013)',
 'Rural-Urban Continuum Code (2023)',
 'Census Region',
 'Census Division',
 'FNS Region']

In [47]:
food_insec.drop('FIPS', axis='columns')

,State,"County, State",Year,Overall Food Insecurity Rate,# of Food Insecure Persons Overall,Food Insecurity Rate among Black Persons (all ethnicities),Food Insecurity Rate among Hispanic Persons (any race),"Food Insecurity Rate among White, non-Hispanic Persons",SNAP Threshold,% FI ? SNAP Threshold,...,% food insecure children in HH w/ HH incomes below 185 FPL,% food insecure children in HH w/ HH incomes above 185 FPL,Cost Per Meal,Weighted weekly $ needed by FI,Weighted Annual Food Budget Shortfall,Rural-Urban Continuum Code (2013),Rural-Urban Continuum Code (2023),Census Region,Census Division,FNS Region
0,AL,"Autauga County, Alabama",2019,15.7%,"8,670",26%,NaN,12%,130%,49%,...,69%,31%,$3.00,$16.88,"$4,439,000",2.0,NaN,South,East South Central,Southeast
1,AL,"Autauga County, Alabama",2020,14.5%,"8,070",25%,NaN,9%,130%,48%,...,71%,29%,$3.22,$17.09,"$4,184,000",2.0,NaN,South,East South Central,Southeast
2,AL,"Autauga County, Alabama",2021,13.3%,"7,770",23%,NaN,9%,130%,48%,...,68%,32%,$3.60,$20.99,"$4,947,000",2.0,NaN,South,East South Central,Southeast
3,AL,"Autauga County, Alabama",2022,15.1%,"8,860",23%,NaN,11%,130%,44%,...,68%,32%,$4.01,$24.86,"$6,680,000",2.0,2.0,South,East South Central,Southeast
4,AL,"Autauga County, Alabama",2023,15.1%,"8,970",24%,NaN,12%,130%,43%,...,66%,34%,$3.64,$22.72,"$6,183,000",2.0,2.0,South,East South Central,Southeast
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
15711,WY,"Weston County, Wyoming",2019,12.2%,860,NaN,NaN,12%,130%,48%,...,62%,38%,$3.21,$18.07,"$471,000",7.0,NaN,West,Mountain,Mountain Plains
15712,WY,"Weston County, Wyoming",2020,11.0%,760,NaN,NaN,10%,130%,49%,...,50%,50%,$3.37,$17.90,"$413,000",7.0,NaN,West,Mountain,Mountain Plains
15713,WY,"Weston County, Wyoming",2021,10.2%,710,NaN,NaN,10%,130%,47%,...,49%,51%,$3.73,$21.74,"$468,000",7.0,NaN,West,Mountain,Mountain Plains
15714,WY,"Weston County, Wyoming",2022,13.6%,940,NaN,NaN,12%,130%,38%,...,47%,53%,$4.13,$25.57,"$729,000",7.0,9.0,West,Mountain,Mountain Plains
